newapi/ooniapi/prio.py

```diff
@@ -133,16 +141,41 @@ def fetch_reactive_url_list(cc: str):
     WHERE probe_cc = :cc
 ) AS cnt
 ON (citiz.url = cnt.input)
-ORDER BY COALESCE(msmt_cnt, 0)::float / GREATEST(priority, 1), RANDOM() ASC
 """
+    # support uppercase or lowercase match
     q = current_app.db_session.execute(sql, dict(cc=cc, cc_low=cc.lower()))
     entries = tuple(q.fetchall())
     log.info("%d entries", len(entries))
-    return entries
+
+    # Fetch url priorities rules
+    sql = """SELECT category_code, cc, domain, url, priority
+    FROM url_priorities WHERE cc = :cc OR cc = '*'
+    """
+    q = current_app.db_session.execute(sql, dict(cc=cc))
```

Comment:
Since you support upper and lowercase match in the citizenlab table, perhaps you should give a similar treatment to the cc match in the `url_priorities` table?

Reply:
No, url_priorities is correctly using uppercase. We might want to switch citizenlab to uppercase in future.